react/components/MadieDialog/index.jsx

```diff
@@ -93,7 +94,7 @@ const Actions = ({ onClose, cancelButtonProps, continueButtonProps }) => {
                             additionalSelectOptionProps={popoverOptions}
                         />
                     </>
-                ) : (
+                ) : continueButtonHide ? null : (
```

Comment:
Seems like Ashok's previous change for hiding the cancel button just keyed off existence of cancelButtonProps. Is that something you already tried and was not possible?

Reply:
@nmorasb I talked with Ashok and made changes to comply with the cancel button hiding changes he made.